crates/bevy_ecs/src/system/function_system.rs

```diff
@@ -211,6 +218,42 @@ impl<Param: SystemParam> SystemState<Param> {
         }
     }
 
+    /// Retrieve the [`SystemParam`] values. This can only be called when all parameters are read-only.
+    /// This will not update archetypes automatically nor increment the world's change tick.
```

Comment:
this line doesn't feel accurate: it's not the archetypes that need to be updated, it's the state view of the world archetypes